test/test_mps.py

```diff
@@ -9657,6 +9656,69 @@ class TestConsistency(TestCase):
         "true_divide"
     }
 
```

Comment:
Seems a bit ugly to me. I don't have a suggestion, but are all these values needed for comparison ?

Reply:
This is better approach and looks cleaner.